Source/Timeline/TimelinePagerView.swift

```diff
@@ -20,7 +20,14 @@ public final class TimelinePagerView: UIView, UIGestureRecognizerDelegate, UIScr
   public weak var dataSource: EventDataSource?
   public weak var delegate: TimelinePagerViewDelegate?
 
-  public private(set) var calendar: Calendar = Calendar.autoupdatingCurrent
+  public var calendar: Calendar = Calendar.autoupdatingCurrent {
+    didSet {
+      pagingViewController.viewControllers?.forEach {
+        let vc = $0 as! TimelineContainerController
```

Comment:
Please use an `if-let` or `guard-let` construct here.

Reply:
Do you think it's worth doing it? After all, `TimelinePagerView` only works with `TimelineContainerController`s, and I think that it is better to have a crash in such a place than to continue working with a controller of a different type.

Reply:
It would be simpler to modify, e.g. to use a protocol instead of a concrete class `TimelineContainerController`.

I'd go for a safe solution without a crash here, although I agree that it's highly unlikely that a crash will happen.